.gitignore

```diff
@@ -1,12 +1,6 @@
 # If a change is made in here, add it to .npmignore as well unless you
 # explicitly want to include it in the published artifact
-node_modules/*
-!node_modules/@okta
-node_modules/@okta/*
-!node_modules/@okta/i18n
-!node_modules/@okta/courage
-!node_modules/@okta/jquery.simplemodal
-!node_modules/@okta/qtip2
+node_modules
```

Comment:
Not really - the old gitignore was basically saying "ignore node_modules except for everything under @okta, except for things in @okta/i18n that are not the json/properties files". The things I copied over are exactly what we want to check in (and if they were changed, would need to be checked into source).